Author: Emilio Pozuelo Monfort <pochu@debian.org>
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=779041
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=837067

--- a/libsecret/test-collection.c
+++ b/libsecret/test-collection.c
@@ -56,6 +56,8 @@
 	test->service = secret_service_get_sync (SECRET_SERVICE_NONE, NULL, &error);
 	g_assert_no_error (error);
 	g_object_add_weak_pointer (G_OBJECT (test->service), (gpointer *)&test->service);
+
+	sleep(1);
 }
 
 static void
@@ -67,6 +69,7 @@
 	g_assert (test->service == NULL);
 
 	mock_service_stop ();
+	sleep(1);
 }
 
 static void
