Index: libsecret-0.16/libsecret/tests/Makefile.am
===================================================================
--- libsecret-0.16.orig/libsecret/tests/Makefile.am	2013-09-17 02:24:05.000658385 +0200
+++ libsecret-0.16/libsecret/tests/Makefile.am	2013-09-17 02:24:05.000658385 +0200
@@ -136,7 +136,7 @@
 test-py:
 	@for py in $(PY_TESTS); do echo "TEST: $$py"; $(PY_ENV) python $(srcdir)/$$py; done
 
-test: test-c test-py test-js $(VALA_TEST_TARGET)
+test: test-c test-py $(VALA_TEST_TARGET)
 
 # ------------------------------------------------------------------
 # INTROSPECTION
