Index: libsecret-0.16/libsecret/Makefile.am
===================================================================
--- libsecret-0.16.orig/libsecret/Makefile.am	2013-09-17 02:24:00.396584851 +0200
+++ libsecret-0.16/libsecret/Makefile.am	2013-09-17 02:24:00.392584788 +0200
@@ -138,10 +138,10 @@
 SecretUnstable_0_gir_FILES = $(UNSTABLE_FILES)
 SecretUnstable_0_gir_SCANNERFLAGS = --c-include "libsecret/secret.h" --identifier-prefix=Secret --symbol-prefix=secret --include-uninstalled=$(builddir)/Secret-@SECRET_MAJOR@.gir
 
-girdir = $(datadir)/gir-1.0
+girdir = $(INTROSPECTION_GIRDIR)
 gir_DATA = $(INTROSPECTION_GIRS)
 
-typelibsdir = $(libdir)/girepository-1.0
+typelibsdir = $(INTROSPECTION_TYPELIBDIR)
 typelibs_DATA = $(INTROSPECTION_GIRS:.gir=.typelib)
 
 if ENABLE_VAPIGEN
